content/news/XX-XX-XXXX-mixxx_2_4_features.md

```diff
@@ -0,0 +1,83 @@
+title: Mixxx 2.4 Release
+authors: Jörg Wartenberg
+status: draft
+tags: release, 2.4
+comments: yes
+
+
+
+## Mixxx 2.4
+
+Mixxx 2.4 is our next major release shipping the work of 2 years continuous development on features, code refacturing and bug fixing.
+
+### EngineOS export
+Mixxx 2.4 adds support for exporting crates, playlists and the library to Engine OS devices ( Denon and Numark standalone controllers).
+This allows users to prepare their tracks on a laptop using Mixxx, export them on an USB stick and plug it to the standalone controller at your gig.
+
+### Saved Loops
+Mixxx 2.4 adds support for Saved loops
+If you have already tracks with Saved Loops defined by Serato DJ, Mixxx 2.4 will automatically read them from the track metadata.
+
+
+### Effects
+Mixxx 2.4 introduces Effect Chains, a set of effects with parameters which can be saved and loaded with all it's parameters. Mixxx 2.4 visualizes these effect parameter by value, unit and knob position.
+
+### Sync
+Mixxx 2.4 adds support for setting an explicit leader for sync. Only one deck can be the sync leader, this is set and indicated by the new crown symbol button.
+This mode is useful if you work with tracks, with variable BPM.
+
+### Track Menu
+The track menu in Mixxx 2.4 is extended by several new features:
+-An action to select loaded track in library
+-An action to analyze/re-analyze with variable/constant BPM on a per-track basis
+-An action for looking up track metadata at Discogs, Soundcoud and LastFM
+-An action to remove track files from disk
+
+### Many other changes
+
+#### Soundstrecher / Key-Shift-Algorithms
+Mixxx 2.4 supports now 3 alternative Key-Shift-Algorithms
+- Soundtouch (Basic audio quality - high performance)
+- Rubberband R2 (High audio quality - higher CPU usage)
+- Rubberband R3 (HIFI audio quality - very high CPU usage)
+
+#### Major rework of the Waveform Rendering code
+Solves some long lasting performance issue, especially on macOS.
+
+#### Native macOS ARM builds for M1/M2 Apple silicon
+
+#### Controller Backend
+ES6 based controller mapping system
```

Comment:
should we mention caveats (such as no module support)?

Reply:
I wouldn't do this here, this shoud be advertizing for the great features in this release - this has no relevance for end users and therefore the media.

Reply:
Fair